spec/exp_descriptor_de_spec.rb

```diff
@@ -385,5 +385,15 @@ def desc(expression, opts = {})
         expect(desc('0 0 0 1 MAR * 2010/5')).to eq('Um 12:00 AM, am 1 Tag des Monats, nur in März, Alle 5 Jahren, beginnend in 2010')
       end
     end
+
+    context 'every few days:' do
+      it 'odd days' do
+        expect(desc('30 7 1-31/2 * *')).to eq('Um 7:30 AM, Alle 2 Tagen, zwischen Tag 1 und 31 des Monats')
+      end
+
+      it 'even days' do
```

Comment:
The test for `even days` seems redundant. I'd drop it.
if you  care about the valid dates I'd go with something general like `1-28/2`, or `1-L/2`.

Reply:
Done 😄